react/features/app/components/Root.web.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+import { App } from './App';
+
+/**
+ * React component for root component.
+ *
+ * @class Root
+ * @extends Component
+ */
+export default class Root extends Component {
```

Comment:
I don't want a `Root` component for Web. The `Root` component for react-native is because of the limitation of react-native. I still want third parties to see in our index.web.js that we're using `App` in a well-known React way.